Clean Code/ToDo/Program.cs

```diff
@@ -1,110 +1,156 @@
+            }
+
+            return userOption;
         }
 
-        public static void ShowMenuDos()
+        private static void ShowMenuAddTask()
+        {
+            Console.WriteLine("Enter the name of the task: ");
+
+            string? task = Console.ReadLine();
+
+            if (task == null || task?.Length <= 0)
+            {
+                Console.WriteLine("No task was added.");
+                return;
+            }
+
+            TaskList?.Add(task);
+            Console.WriteLine("Task registered successfully");
+        }
+
+        private static void ShowMenuRemoveTask()
         {
             try
             {
+                if (TaskList?.Count <= 0)
+                {
+                    Console.WriteLine("There are no tasks to remove.");
+                    return;
+                }
+
                 Console.WriteLine("Enter the number of the task to remove: ");
                 // Show current taks
-                for (int i = 0; i < TL.Count; i++)
+                for (int i = 0; i < TaskList?.Count; i++)
                 {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
+                    int j = i + 1;
+
+                    Console.WriteLine(j + ". " + TaskList[i]);
                 }
                 Console.WriteLine("----------------------------------------");
 
-                string line = Console.ReadLine();
+                string? line = Console.ReadLine();
+                if (line?.Length <= 0)
+                {
+                    Console.WriteLine("No task was removed.");
+                    return;
+                }
+
                 // Remove one position
                 int indexToRemove = Convert.ToInt32(line) - 1;
+
                 if (indexToRemove > -1)
                 {
-                    if (TL.Count > 0)
-                    {
-                        string task = TL[indexToRemove];
-                        TL.RemoveAt(indexToRemove);
-                        Console.WriteLine("Task " + task + " deleted");
-                    }
+                    string task = TaskList[indexToRemove];
+                    TaskList.RemoveAt(indexToRemove);
+
+                    Console.WriteLine("Task " + task + " deleted");
                 }
             }
-            catch (Exception)
+            catch (Exception ex)
             {
+                Console.WriteLine($"RemoveTask failed. ({ex.Message})");
             }
         }
 
-        public static void ShowMenuAdd()
+        private static void ShowMenuPendingTask()
         {
             try
             {
-                Console.WriteLine("Enter the name of the task: ");
-                string task = Console.ReadLine();
-                TL.Add(task);
-                Console.WriteLine("Task registered successfully");
-            }
-            catch (Exception)
-            {
-            }
-        }
+                if (TaskList == null || TaskList.Count == 0)
```

Comment:
Please follow the KISS principle and refactor this logic 
```suggestion
                if (TaskList?.Count > 0)
                  // logic to show the task list
                else
                  // logic when there are no tasks
```